.gitignore

```diff
@@ -5,4 +5,7 @@ NinchatSDKSwift.xcodeproj/xcuserdata/
 NinchatSDKSwift.xcworkspace/xcuserdata/
 NinchatSDKSwiftServerTests/Secrets.plist
 .idea/
+
+open-dev-env.sh
+close-dev-env.sh
```

Comment:
You don't need to use the committed `.gitignore` to enumerate all files which you might have in your working directory; you can use `.git/info/exclude` per repository, or specify a global ignore file by adding this to your `~/.gitconfig`:

```
[core]
        excludesfile = <path>
```

Reply:
Oh, very good point 🤔🤔

Reply:
I added a new directory path to `.git/info/exclude`